app/models/submission.rb

```diff
@@ -1,6 +1,7 @@
 class Submission < ActiveRecord::Base
   belongs_to :student
   belongs_to :repo
+  belongs_to :user
```

Comment:
👍 I had to add an additional foreign key field since it wouldn't create the relationship independently of that. It maybe have been a result of the way I set up the table in the migration